doc/externalGrading.md

```diff
@@ -11,7 +11,7 @@ You can define a number of resources for the external grading process:
 * Files that are shared between questions
 * Tests for individual questions
 
-When a student clicks "Submit" on an externally-graded question, PrairieLearn will assemble all of those resources that you've defined into an archive. That archive will be submitted to be run as a job on AWS infrasctructure inside the environment that you specify. Results are then sent back to PrairieLearn; once they are processed, the student can refresh the page to see their score, individual test cases, `stdout`/`stderr`, and more.
+When a student clicks "Submit" on an externally-graded question, PrairieLearn will assemble all of those resources that you've defined into an archive. That archive will be submitted to be run as a job on AWS infrastructure inside the environment that you specify. Results are then sent back to PrairieLearn; once they are processed, the student will immediately be show their score, individual test cases, `stdout`/`stderr`, and any information you want to show to them.
 
 ## External grading timestamps and phases
```

Comment:
This feels out of place here - not sure where this should actually go.

Reply:
Agreed. I just stuck it somewhere so we've have it written down. Feel free to shift it.